web3swift/Web3/Classes/Web3+JSONRPC.swift

```diff
@@ -24,10 +24,10 @@ public struct Counter {
 
 /// JSON RPC request structure for serialization and deserialization purposes.
 public struct JSONRPCrequest: Encodable {
-    var jsonrpc: String = "2.0"
-    var method: JSONRPCmethod?
-    var params: JSONRPCparams?
-    var id: UInt64 = Counter.increment()
+    public var jsonrpc: String = "2.0"
+    public var method: JSONRPCmethod?
+    public var params: JSONRPCparams?
+    public var id: UInt64 = Counter.increment()
```

Comment:
I understand that public constructor for a “Dispatcher” may be necessary, but request should not be messed up with (or please provide a concrete example for what reason). May be extending a list of JSONRPCmethod is better approach?

Reply:
I want to integrate web3 with native wallet app and need to read method/params/id to determine whether to intercept the request. For example, I want to intercept eth.sendTransaction request and handle it by myself, so I need to know `method` first and then use `id` / `params` to process the request.